docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -0,0 +1,133 @@
+
+### Traces
+
+Traces give an overview of what is happening in a specific section of the execution of an application. A trace is composed by a hierarchical sequence of segments, each of which contain basic information about the execution (e.g., duration), and whether that section was executed successfully or not.
+
+In the AWS JDBC Driver, initially a trace will be generated for every JDBC call made to the wrapper. Depending on whether the user application has already a trace open, it might be either nested into it or dropped. And then, for each plugin that is part of the wrapper plugin chain, another segment will be created only for the plugin execution, linked to the JDBC call segment.
+
+Traces from the AWS JDBC Driver are submitted to **AWS X-Ray**. [link]
+
+### Metrics
+
+Metrics are numeric data that were measured and collected through the execution of an application. Those metrics can give an insight on how many times some action (e.g., failover) has happened, and if that action succeeded or failed, and other related information.
+
+The AWS JDBC Driver will submit metrics data to **AWS Cloudwatch**.
+
+The list of available metrics available for the AWS JDBC Driver and its plugins is available in the Appendix [link] section of this page.
+
+## Setting up the Amazon OTEL Collector
+
+## Pre Requirements
+
+Before enabling the Telemetry feature, a few setup steps are required to ensure the monitoring data gets properly emitted.
+
+1. In order to visualize the telemetry data in the AWS Console, make sure you have an IAM user or role with permissions to AWS X-Ray and AWS Cloudwatch.
+
+2. Download the AWS OTEL Collector and set it up. The AWS OTEL Collector is responsible from receiving telemetry data from the application using the AWS JDBC Driver and forward it to AWS. Both of those connections happen via HTTP, therefore URLs and ports need to be correctly configured for the collector.
+
+WARNING: The AWS OTEL Collector can be set up either locally or remotely. It is up to the user to decide where is best to set it up. If you decide to host it remotely, ensure that the application has the necessary permissions or allowlists to connect to the Collector.
+
+WARNING: The collector is an external application that is not part of the wrapper itself. Without a collector, the wrapper will collect monitoring data from its execution but that data will not be sent anywhere for visualization.
+
+## Using Telemetry
+
+Telemetry for the AWS JDBC Driver is a monitoring strategy that overlooks all plugins that are enabled in [`wrapperPlugins`](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters) and is not a plugin in itself. Therefore no changes are required in the `wrapperPlugins` parameter to enable Telemetry.
+
+In order to enable Telemetry in the AWS JDBC Driver all you need to do is to set the `enableTelemetry` property to `true` either in the code or in the connection URL.
```

Comment:
Can we provide examples for both and for datasources ? So users can just copy pasta?

Reply:
Nit: can we use connection string instead of connection URL